understanding/focus-not-obscured-minimum.html

```diff
@@ -114,7 +114,7 @@ <h2>意図</h2>
                </p>
                <p>フォーカスされた項目と重なる可能性のある代表的なコンテンツの種類は、スティッキーフッター、スティッキーヘッダー、及び非モーダルダイアログである。利用者がページ内をタブで移動すると、これらのコンテンツのレイヤーにより、フォーカスを受け取った項目がフォーカスインジケータと共に隠されてしまう可能性がある。
```

Comment:
「非モーダルダイアログ」ですが、「非」が「モーダル」「ダイアログ」どちらにかかっているのかが一見わかりにくい感じがしたので、「モーダル」にかかることをわかりやすくするために (つまりモードレスダイアログであることが伝わるように)、「モーダルではないダイアログ」とするのはいかがでしょうか？
```suggestion
               <p>フォーカスされた項目と重なる可能性のある代表的なコンテンツの種類は、スティッキーフッター、スティッキーヘッダー、及びモーダルではないダイアログである。利用者がページ内をタブで移動すると、これらのコンテンツのレイヤーにより、フォーカスを受け取った項目がフォーカスインジケータと共に隠されてしまう可能性がある。
```